### PR TITLE
Add extensions requested to latest versions.

### PR DESCRIPTION
**Pull request by UserNotFound**
Add plperl and pg_repack up to version 13

### CODE REVIEW

File: 12-contrib/install-extras.sh [hunk 1]
```diff
@@ -11,7 +11,8 @@ apt-key add /tmp/GPGkeys/pglogical.key
 # Install packaged extensions first
 apt-install "^postgresql-plperl-${PG_VERSION}$" "^pgagent$" "^postgresql-${PG_VERSION}-pglogical$" \
   "^postgresql-plpython3-${PG_VERSION}$" "^postgresql-${PG_VERSION}-wal2json$" \
-  "^postgresql-${PG_VERSION}-pgaudit$"
+  "^postgresql-${PG_VERSION}-pgaudit$" "^postgresql-plperl-${PG_VERSION}$" \
```
Comment (bayley-aptible) [inline 12-contrib/install-extras.sh:14]: plperl ended up in here twice

**Review by bayley-aptible — APPROVED** [review]
LGTM w/ one item to clean up!

File: 12-contrib/install-extras.sh (hunk 1 above)
Comment (UserNotFound) [reply 12-contrib/install-extras.sh]: Thanks!

**Comment by UserNotFound** [comment]
Customer didn't actually need these.

**Pull request closed without merging**